.github/workflows/ci-backend.yml

```diff
@@ -111,28 +111,65 @@ jobs:
         needs: changes
         if: ${{ needs.changes.outputs.backend == 'true' && github.repository == 'PostHog/posthog' }}
 
-        name: Django tests – Py ${{ matrix.python-version }}
+        name: Django tests – Py ${{ matrix.python-version }} ${{ matrix.name }} (${{matrix.group}}/${{ matrix.total-concurrency }})
         runs-on: ubuntu-latest
 
         strategy:
             fail-fast: false
             matrix:
-                python-version: ['3.8.5', '3.9.0']
+                python-version: ['3.8.5']
+                ee: [true]
+                foss: [false]
+                saml: [false]
+                name: ['']
+                total-concurrency: [5]
+                group: [1, 2, 3, 4, 5]
+                include:
+                    # :TRICKY: Run FOSS tests in a separate container
+                    - python-version: '3.8.5'
+                      ee: false
+                      saml: false
+                      foss: true
+                      name: 'FOSS'
```

Comment:
Is it worth pulling this name up to the top e.g. `name: ['ee', 'foss', 'saml']` and then using include to specific options for each? Might make a little easier to read what's going on?

Reply:
This wouldn't work because then we run N x M tests.

Even if it did work I don't see it being clearer since now you need to do string comparison across the many conditionals below which is IMO worse.

Reply:
Ok. Not sure I understand all that but I'm not so familiar with github matrix. I assumed that:

```
name: ['ee', 'foss', 'saml']
include:
   - name: ee
      total-concurrency: 5
   - name: foss
      total-concurrency: 1
...
```

would work, but perhaps not.